tests/safety_replay/helpers.py

```diff
@@ -58,17 +57,21 @@ def set_desired_torque_last(safety, mode, torque):
   elif mode == Panda.SAFETY_SUBARU:
     safety.set_subaru_desired_torque_last(torque)
 
-def package_can_msg(msg):
-  rdlr, rdhr = struct.unpack('II', msg.dat.ljust(8, b'\x00'))
-
-  ret = libpandasafety_py.ffi.new('CAN_FIFOMailBox_TypeDef *')
-  if msg.address >= 0x800:
-    ret[0].RIR = (msg.address << 3) | 5
+def len_to_dlc(length):
```

Comment:
why not use the `DLC_TO_LEN` LUT you made in `__init__.py`?

Reply:
and maybe the packaging of the msg should also just use a helper function in `__init__.py` to avoid redundance.
Can also add a unit test to make sure `unpack(pack(msg)) == msg`

Reply:
It has different logic, but discussed with you already, will change this approach